cypress/package.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Not sure if we need a separate package.json. Are you implying we should add commands to the main package.json that allow running tests from the root directory?

Reply:
I think at this time I am going to lean towards keeping them separate. While we haven't decided if cypress is our permanent path forward, I would rather not consolidate as it will be harder to identify all the components to remove if we choose a framework other than cypress for the long-term.

Reply:
I think we should be adding cypress to the frontend package.json as a devDependency. That way we can make sure that our testing framework is aligned and using the same versions as our frontend for things like cloudscape, eslint, or even typescript.

Reply:
if we are holding off until we can determine if we want to continue using cypress at all, then maybe we should decide this before pushing these changes to main.